engine/common/components-tests/src/test/scala/pl/touk/nussknacker/engine/common/components/DecisionTableSpec.scala

```diff
@@ -0,0 +1,162 @@
+package pl.touk.nussknacker.engine.common.components
+
+import cats.data.ValidatedNel
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.engine.api.component.ComponentDefinition
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError
+import pl.touk.nussknacker.engine.build.{GraphBuilder, ScenarioBuilder}
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.engine.flink.test.FlinkSpec
+import pl.touk.nussknacker.engine.flink.util.test.FlinkTestScenarioRunner
+import pl.touk.nussknacker.engine.lite.util.test.LiteTestScenarioRunner._
+import pl.touk.nussknacker.engine.flink.util.test.FlinkTestScenarioRunner._
+import pl.touk.nussknacker.engine.graph.expression.Expression
+import pl.touk.nussknacker.engine.lite.util.test.LiteTestScenarioRunner
+import pl.touk.nussknacker.engine.spel
+import pl.touk.nussknacker.engine.util.test.{RunListResult, TestScenarioRunner}
+import pl.touk.nussknacker.test.ValidatedValuesDetailedMessage
+
+import java.util.{List => JList, Map => JMap}
+import scala.jdk.CollectionConverters._
+import scala.reflect.ClassTag
+
+trait DecisionTableSpec extends AnyFreeSpec with Matchers with ValidatedValuesDetailedMessage {
+
+  import spel.Implicits._
+
+  "Decision Table component should" - {
+    "filter and return decision table's rows filtered by the expression" in {
```

Comment:
It would be nice to have more tests like: >1 record, accessing the column that is not available nad so on, but I'm not forcing to write them :)